src/services/pokemon.js

```diff
@@ -50,9 +50,9 @@ export const fetchSearchPokemon = (pokemonName) => {
 };
 
 export const fetchTypes = async () => {
```

Comment:
Another function in this file needs to be async as well. Remember that anytime you fetch from an external source, you'll need to do it asynchronously